controllers/utils_test.go

```diff
@@ -220,3 +221,12 @@ func randStringRunes(n int) string {
 func newName(prefix string) string {
 	return fmt.Sprintf("%s-%s", prefix, randStringRunes(8))
 }
+
+func policyServerPodDisruptionBudgetExists(policyServerName string) error {
+	podDisruptionBudgetName := fmt.Sprintf("policy-server-%s-%s", policyServerName, "pdb")
```

Comment:
nitpicking: I think we can omit `pdb` from the final PDB name, it's a bit redundant

Reply:
Also, we have a function that computes that name: `buildPodDisruptionBudgetName`. It should be used